apps/server/src/api.ts

```diff
@@ -0,0 +1,402 @@
+import cors from "cors";
+import express from "express";
+import { randomUUID } from "node:crypto";
+import { fileURLToPath } from "node:url";
+import { data as rawData } from "./data";
+
+import type { PagedData, Role, User } from "./models";
+
+const serverConfig = {
+  port: +(process.env.SERVER_PORT || 3001),
```

Comment:
One small thing to note, I wasnt supposed to update the server package, but I did make this one change (sorry!)

- Client is served on 3000
- Server is served on 3001

Willing to revert this if need be, it was just a small thing that kind of irked me.